install/suse_dependencies.sh

```diff
@@ -62,7 +62,7 @@ pip install --upgrade pyparsing
 #apt-get install -y ipython-notebook
 pip install --upgrade ipython
 pip install --upgrade jupyter
-pip install --upgrade matplotlib
+pip install --upgrade matplotlib==2.0.2
```

Comment:
Because basemap will throw depreciation warnings if matplotlib is too new. There was talk about moving away from basemap so I figured this would be a quick fix.